ivadomed/transforms.py

```diff
@@ -3,7 +3,6 @@
 import math
 import numbers
 import random
-
```

Comment:
Please revert this change. There should be no change here.

Reply:
fixed.